[2025-04-18T08:19:12.358+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T08:19:12.372+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ssh_to_host_with_paramiko.execute_remote_script_on_host manual__2025-04-18T08:16:12.616522+00:00 [queued]>
[2025-04-18T08:19:12.378+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ssh_to_host_with_paramiko.execute_remote_script_on_host manual__2025-04-18T08:16:12.616522+00:00 [queued]>
[2025-04-18T08:19:12.378+0000] {taskinstance.py:2867} INFO - Starting attempt 3 of 3
[2025-04-18T08:19:12.389+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): execute_remote_script_on_host> on 2025-04-18 08:16:12.616522+00:00
[2025-04-18T08:19:12.393+0000] {standard_task_runner.py:72} INFO - Started process 493 to run task
[2025-04-18T08:19:12.395+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ssh_to_host_with_paramiko', 'execute_remote_script_on_host', 'manual__2025-04-18T08:16:12.616522+00:00', '--job-id', '332', '--raw', '--subdir', 'DAGS_FOLDER/executeremote.py', '--cfg-path', '/tmp/tmp7mo7chkp']
[2025-04-18T08:19:12.397+0000] {standard_task_runner.py:105} INFO - Job 332: Subtask execute_remote_script_on_host
[2025-04-18T08:19:12.437+0000] {task_command.py:467} INFO - Running <TaskInstance: ssh_to_host_with_paramiko.execute_remote_script_on_host manual__2025-04-18T08:16:12.616522+00:00 [running]> on host 539fb9046bc5
[2025-04-18T08:19:14.171+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ssh_to_host_with_paramiko' AIRFLOW_CTX_TASK_ID='execute_remote_script_on_host' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T08:16:12.616522+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-18T08:16:12.616522+00:00'
[2025-04-18T08:19:14.172+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-18T08:19:14.217+0000] {transport.py:1944} INFO - Connected (version 2.0, client OpenSSH_9.6p1)
[2025-04-18T08:19:14.236+0000] {transport.py:1944} INFO - Authentication (publickey) successful!
[2025-04-18T08:19:14.525+0000] {logging_mixin.py:190} INFO - STDOUT: hello world
[2025-04-18T08:19:14.526+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-18T08:19:14.537+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-18T08:19:14.537+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ssh_to_host_with_paramiko, task_id=execute_remote_script_on_host, run_id=manual__2025-04-18T08:16:12.616522+00:00, execution_date=20250418T081612, start_date=20250418T081912, end_date=20250418T081914
[2025-04-18T08:19:14.575+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T08:19:14.589+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-18T08:19:14.591+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
